### PR TITLE
Add traverse function

### PR DESCRIPTION
**Pull request by Elaray1**

### CODE REVIEW

**Comment by AllDayAlone** [comment]
@shevv920 what do you think about this?

File: src/traverseFunc.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var _ = require('lodash');
+
+export const traverse = (obj, func) => {
+    _.forIn(obj, (val, key) => {
+        obj[key] = func(val);
+        if (_.isArray(val)) {
```
Comment (shevv920) [inline src/traverseFunc.js:6]: _.isArray could be replaced by Array.isArray

**Comment by shevv920** [comment]
> @shevv920 what do you think about this?
Depends 
Is lodash forbidden here?

**Comment by shevv920** [comment]
create issues for used lodash methods, and pretend they are exists

**Pull request merged**